addons/website/views/snippets/snippets.xml

```diff
@@ -1060,6 +1060,80 @@
         </we-button-group>
     </div>
 
+    <div data-js="ConditionalVisibility" data-option-name="conditionalVisibility" data-selector="section">
+        <we-collapse>
+            <we-select string="Visibility" data-attribute-name="visibility" data-no-preview="true">
+                <we-button data-select-data-attribute="">Always visible</we-button>
+                <we-button data-select-data-attribute="conditional" data-select-class="o_snippet_invisible" data-name="visibility_conditional">Conditionally</we-button>
+            </we-select>
+
+            <we-select t-if="request.session.geoip.get('country_code')" 
+                string="⌙ Country" 
+                data-attribute-name="visibilityValueCountryRule" 
+                data-no-preview="true" 
+                data-dependencies="visibility_conditional"
+            >
+                <we-button data-select-data-attribute="">Visible for</we-button>
+                <we-button data-select-data-attribute="hide">Hidden for</we-button>
+            </we-select>
+            <we-many2many t-if="request.session.geoip.get('country_code')"
+                            string=" " 
+                            data-dependencies="visibility_conditional" 
+                            data-attribute-name="visibilityValueCountry" 
+                            data-no-preview="true" 
+                            data-model="res.country" data-call-with="code" data-select-record="" data-fields="[&quot;code&quot;]" 
+                            data-allow-delete="true" data-fakem2m="true"/>
```

Comment:
This part of the UI is kinda repeated 4x times with a few different attributes -> make it a parametrized template that you can t-call 4x here, for each visibility condition part. For the t-if, you should not duplicate it but wrap the two widgets in a 
`<t t-if="..."></t>` (well, wrapping the t-call after the change I am asking here :wink:)